include/swift/ABI/Metadata.h

```diff
@@ -3759,7 +3777,9 @@ class TargetClassDescriptor final
     /// positive size of metadata objects of this class (in words).
     uint32_t MetadataPositiveSizeInWords;
 
-    // Maybe add something here that's useful only for resilient types?
+    /// Classes with resilient ancestry instead use this flags word to
+    /// indicate the presence of an Objective-C resilient class stub.
```

Comment:
It looks like this field should always be present and zero in previously-emitted class descriptors, so this should be ABI-compatible.

Consider just saying "otherwise" instead of spelling out a different condition which an uninitiated reader might not understand is the inverse of "does not have a resilient superclass".

"word" is generally used for a pointer-sized chunk in Swift.  You could just "these flags".

Everything after "to" in this comment is over-specific to your patch.  You can say something like "to do things like indicating..." if you think the example is important to reference.

Reply:
I'll tweak the comment a bit.